scripts/collector/postgres/collect-data.sh

```diff
@@ -37,7 +37,7 @@ GREP=$(which grep)
 SED=$(which sed)
 MD5SUM=$(which md5sum)
 MD5COL=1
-
+ 
```

Comment:
pkey is not sent to postgresql sql files.

Reply:
Added this.